$OpenBSD: patch-configure,v 1.7 2011/03/21 08:04:40 jasper Exp $

force disable Cocoa checks, which will later cause the build to fail
    if gnustep-gui has been installed.
with_internal_maximum_log_level :
	replace \+ by + as it should be
	add -E option to grep, handle the + in expression

--- configure.orig	Mon Oct 31 17:03:43 2011
+++ configure	Thu Nov  3 10:56:21 2011
@@ -14783,7 +14783,7 @@ with_max_log_level="<unset>"
 if test "${with_internal_maximum_log_level+set}" = set; then :
   withval=$with_internal_maximum_log_level;
     if test "x${withval}" != "xno"; then
-       if echo "${withval}" | grep '^[0-9]\+$' >/dev/null 2>/dev/null; then
+       if echo "${withval}" | grep -E '^[0-9]+$' >/dev/null 2>/dev/null; then
           { $as_echo "$as_me:${as_lineno-$LINENO}: ignoring any EINA_LOG() with level greater than ${withval}" >&5
 $as_echo "$as_me: ignoring any EINA_LOG() with level greater than ${withval}" >&6;}
 
@@ -25453,7 +25453,7 @@ fi
 
 cocoa_ldflags="";
 have_cocoa="no"
-
+if false; then # Disable Cocoa checks
     if test "x${have_gnu_objc}" = "xyes" ; then
        ac_ext=m
 ac_cpp='$OBJCPP $CPPFLAGS'
@@ -25612,7 +25612,7 @@ ac_link='$CC -o conftest$ac_exeext $CFLAGS $CPPFLAGS $
 ac_compiler_gnu=$ac_cv_c_compiler_gnu
 
     fi
-
+fi
 
 
 want_epoll=yes
